r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
@13steinj You might want to look at [this comment](https://www.reddit.com/r/OutOfTheLoop/comments/3lj1ez/whats_the_story_behind_create_morrisganis/cv6z0pv) by /u/13steinj.